indigo_api/models.py

```diff
@@ -458,33 +458,33 @@ def friendly_type(self):
         if plugin:
             return plugin.work_friendly_type(self)
 
-    def amendments_with_initial(self):
+    def expressions_by_date(self):
```

Comment:
This doesn't seem right to me since it's not actually returning expressions, it's a lot more than that (contrast with the `expressions` method). Maybe `expression_dates` or `possible_expression_dates`. Neither are great :/